nvidia_entrypoint.sh

```diff
@@ -69,7 +69,14 @@ else
   fi
 fi
 
+avx=1
```

Comment:
Can you revert all changes in this file. The avx and isarm dereferences need '$' but we will fix this file ourselves once we officially support SBSA.